test/Serilog.Tests/Serilog.Tests.csproj

```diff
@@ -1,12 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <!-- Tests target .NET Framework 4.6.2 plus the latest RTM of .NET Framework -->
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT'">net48;net462</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT'">net48;net472</TargetFrameworks>
```

Comment:
Nit: comment above is now outdated.

It's a shame to have to stop testing on 4.6.2 but I'm assuming it's somewhat inevitable at this stage. No way around this while updating to v3?

Reply:
Unfortunately xunit dropped support of net462.